### DISCUSSION TITLE
Argocd Upgrade path

### DISCUSSION THREAD

**Discussion by rajeshnair** [argoproj/argo-cd#10683] [Q&A]
We have been using an argocd deployment 2.0.5 with some 80+ applications deployed.

We now need to upgrade to latest 2.4.0 and I have been trying to find a safe upgrade path documentation but could not find it.

The [upgrading documentation](https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/overview/) simply point to how to install a specific version but does not document how would upgrading impact the existing applications. 

So here are my questions
1. What is the best way to upgrade argocd without impacting existing applications running in it?
2. Is it possible/safe/recommended to deploy newer version of argocd on a new cluster and point gitops repo to that ?
3. How can I export all applications/configurations of existing argocd deployment ?

**Comment by jcmejias1** [comment]
bump to this, I have been able to "upgrade" after jumping a lot of hoops, from v2.4.13 to v2.5.2 but I still see some pots showing the image v2.4.13 as well as the web interface
